bitnami/mysql/Chart.yaml

```
@@ -25,4 +25,4 @@ name: mysql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mysql
   - https://mysql.com
-version: 9.5.1
+version: 9.5.2
```

You are adding a new feature. According to [semver](https://semver.org/) you should bump the minor version
```suggestion
version: 9.6.0
```